src/__test__/utils.test.js

```diff
@@ -561,4 +561,28 @@ describe('test that makeCommentBody makes a nicely-formatted string', () => {
             "
         `);
     });
+
+    it('should comment out reviewers', async () => {
```

Comment:
Is there an equivalent test to check that they are tagged instead of commented when the flag is not true?